config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
+  root to: 'tasks#index'
+  resources :tasks do
```

Comment:
check to make sure there are no duplicate routes, keep it dry: 
(destroy can go outside the loop)
#resources :tasks, only: [:index, :show] do